src/app/containers/Timestamp/index.jsx

```diff
@@ -29,22 +30,60 @@ const formatTimestamp = dateObj => {
   return `${dayNumeric} ${monthLong} ${fullYear}`;
 };
 
-const TimestampContainer = ({ timestamp }) => {
-  const dateObj = new Date(timestamp);
+const isTenHoursAgo = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds >= 10 * 60 * 60 * 1000;
+};
+
+const timestampWithPrefixUpdated = (datetime, milliseconds) => (
+  <Timestamp datetime={datetime} prefix="Updated">
```

Comment:
Not wholly convinced about the `prefix` attribute, but I see it was added in https://github.com/bbc/simorgh/pull/1316 and I understand it was to keep a SEO-semantic `<time>` element in the output.

The concerns outlined in https://github.com/bbc/simorgh/pull/1316#issuecomment-465502868 are to do with the idea that anything inside a `<time></time>` element that is not really a "time" is invalid. But it's not invalid, provided we set a `datetime` attribute, as this takes priority over whatever the contents of `<time></time>` are:

> The datetime value of a time element is the value of the element’s datetime content attribute, if it has one, otherwise the concatenation of the contents of all the Text nodes that are children of the time element (ignoring any other nodes such as comments or elements), in tree order.

(~https://www.w3.org/TR/html51/textlevel-semantics.html#the-time-element)

This decision has also meant that VoiceOver reads out the time twice.

Think we can kill two birds with one stone by changing the API.

```jsx
<Timestamp datetime={datetime}>
    Updated, {convertToHumanReadableFormat(datetime)}
</Timestamp>
```

Reply:
Let's ignore this point for this PR, but refactor it later: https://github.com/bbc/simorgh/issues/1446